Templates/Specimens_Metadata_2021Q1_template.tsv

```diff
@@ -0,0 +1,18 @@
+Column Name	Description	Ontology identifier	Ontology term name	Ontology Description
+Specimen ID	Unique identifier for this specimen	OBI:0100051	specimen	A material entity that has the specimen role.
+Parent specimen ID	Identifier for this specimen’s parent specimen (optional)	OBI:0100051	specimen	A material entity that has the specimen role.
+Number of subspecimens	A count of specimens derived from this specimen (cumulative)	APOLLO_SV:00000032	count	A measurement datum that is the output of counting.
+Specimen type	Type of specimen (excised tissue block, slice, cell, etc.)	AIO:specimen type name	specimen type name	A textual entity that denotes the type of a specimen.
+Hemisphere	Hemisphere that this specimen came from (left, right, both, unknown)	PATO:0000140	position	A spatial quality inhering in a bearer by virtue of the bearer's spatial location relative to other objects in the vicinity.
+Anatomical structure	Specimens location within anatomical structures from brain atlas	PATO:0000141	structure	A morphology quality inhering in a bearer by virtue of the bearer's relative position, shape, arrangements and connectivity of an organism's various parts; the pattern underlying its form.
+Specimen disease condition	Description of specimen’s disease state (healthy vs diseased tissue)	OBI:0000278	disease stage	A part of an occurrence of a disease process which is associated with position in the normal progression of the disease
```

Comment:
Might want a bit of additional explaination that this is NOT the parent specimen disease state. For example.
Patient has diagnosis Alzheimer's, bulk tissue has pathology a beta plaques, etc.

Reply:
I changed the column name to `Specimen pathology`. I'm still not exactly which ontology term to use here and exactly what values this column should have.

Reply:
At the simplest level, I think it would be something like `pathological formation` -- true/false. [tissue has some pathological formation] Does that sound right, @tgbugs ?

We can be more specific, of course, but let's try to agree on this pattern first.

Reply:
I think `specimen pathology` is ok. The tricky bit with trying to use `pathological formation` is that it misses the fact that it has to have been diagnosed. So there might well be a pathological formation in the tissue, but if it was not reported then it would not be able to show up in this field. More importantly, we want to be able to capture the fact that it is possible for an expert pathologist to declare that tissue is indeed normal, and that there are no pathological formations anywhere in the tissue, which is different than "we assumed normal but didnt' look." So we're back to the problem where we have `didn't check` or `assumed normal` (hemisphere `unspecified`) vs `normal` or some `pahtology x`. The exact modelling of tissues that have parts that are somehow pathological is a bit beyond the scope of this I imagine. `neurofibrillary tangles` or `plaques` would probably classify as formations rather than generic pathologies because they are the realization of the disposition latent in some disease (I think). I can't recall right now what the realized disposition is supposed to be for a disease, but the formation probably works. hasPathology o hasPart some pathological formation or something like that (as written that is bad owl)?

Reply:
Yes, I see the epistemological point again. I'm okay with `specimen pathology` instead given these concerns. 

I've been operating on disease as a disposition to undergo a pathological process that exists in an organism due to one or more disorders in that organism. So, Alzheimer's (disease is_a disposition) to undergo a pathological process (amyloid beta protein aggregation) due to (some amyloid plaque). I'm sure we'll get there soon.

Reply:
Sounds good. As I realized when writing up the bit below, pathology might also be defined by the absence of some normal feature, rather than the presence of some pathological feature, which is another reason why `specimen pathology` seems like the right way to go.

As a side note the `due to` is pointing in the wrong direction here I think. The pathological process produces the plaques and the plaques are evidence of some pathological process, but the immediate `due to` is a combination of at least two different causes, increased production and/or decreased clearing, with a possible third and fourth being increased aggregation and/or decreased disaggregation. The underlying causes for those are mostly open questions, but the presence of the plaques is the pathology (in other cases the absence of an otherwise present feature could also be a pathology, so the presence/absence has to be factored in, re: coffee without milk).

Reply:
Yes, absence of a feature should be counted as a disorder, I think. 

> The pathological process produces the plaques and the plaques are evidence of some pathological process, but the immediate due to is a combination of at least two different causes, increased production and/or decreased clearing, with a possible third and fourth being increased aggregation and/or decreased disaggregation. The underlying causes for those are mostly open questions, but the presence of the plaques is the pathology (in other cases the absence of an otherwise present feature could also be a pathology, so the presence/absence has to be factored in, re: coffee without milk).

Good catch -- it is some other disorder that leads to the pathological process that realizes the disposition. Those plaques may be _a_ result of the pathological process, but are not the disorder that leads to the pathological process. That is to say, some genetic disorder would lead to a pathological process that realizes the disease -- perhaps `tumor` is the analogous example here.